pkg/asset/manifests/azure/cluster.go

```diff
@@ -381,6 +381,16 @@ func getNextAvailableIPForLoadBalancer(ctx context.Context, installConfig *insta
 		}
 		machineCidr = cidrRange
 	}
+	// AzureStack does not support the call to CheckIPAddressAvailability.
```

Comment:
This uses the 5th IP, which makes sense to me. Wondering if we have existing job to test this?

Reply:
This is an untested fix for the BYO network scenario.

Reply:
Oh okayy! Would be nice to have one later one ^-^

Reply:
@jinyunma confirmed that this works for installing a cluster into BYO vnet. We then hit another issue when provisioning the NSG, which should now be resolved.

There is a further bug that a second cluster cannot be installed into the vnet. That should be possible to workaround by editing manifests and we can handle subsequently.